suites/squid/common/regression/nvme-single-site-deploy-and-configure.yaml

```diff
@@ -0,0 +1,170 @@
+# Test suite deploy and configure single site cluster
+# global-conf: conf/squid/common/17node-4client-single-site-regression.yaml
+
+tests:
+  - test:
+      name: Setup pre-requisites
+      desc: Setup packages and configuration for cluster deployment
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: Deploy cluster using cephadm
+      desc: Bootstrap and deploy services
+      polarion-id: CEPH-83573713
+      module: test_cephadm.py
+      config:
+        steps:
+          - config:
+              service: cephadm
+              command: bootstrap
+              args:
+                mon-ip: node1
+          - config:
+              service: host
+              command: add_hosts
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply_spec
+              service: orch
+              validate-spec-services: true
+              specs:
+                - service_type: osd
+                  service_id: all-available-devices
+                  encrypted: "true"
+                  placement:
+                    nodes:
+                      - node2
+                      - node3
+                      - node4
+                      - node5
+                      - node6
+                      - node7
+                      - node8
+                      - node9
+                      - node10
+                      - node11
+                      - node12
+                      - node13
+                  spec:
+                    data_devices:
+                      all: "true"
+                    encrypted: "true"
+          - config:
```

Comment:
I think this config step has to be removed

Reply:
@AmarnatReddy can you confirm?

Reply:
done